core/components/form-fields/utils.ts

```diff
@@ -47,6 +47,8 @@ export const BOTH_CUSTOMER_ADDRESS_FIELDS = [
   FieldNameToFieldId.phone,
 ];
 
+export const NAME_FORM_FIELD = [FieldNameToFieldId.firstName, FieldNameToFieldId.lastName];
```

Comment:
what do you think name to `FULL_NAME_FIELDS`?

Reply:
@bc-alexsaiannyi changed, thanks for the suggestion!